index.js

```diff
@@ -34,13 +34,14 @@ var gitlab = require('gitlab')((function () {
   };
 
   if (!options.url) {
-    var question = 'Enter GitLab URL';
     var defaultInput = (function () {
       var url = git.config.get('remote.origin.url');
-      return url && 'https://' + gitUrlParse(url).resource;
+      if (!url || url.indexOf('bitbucket') !== -1 || url.indexOf('github') !== -1) {
+        url = 'https://gitlab.com';
```

Comment:
In which case we will need this. Your `origin` url is something other than gitlab. Wouldn't user just put it at the prompt. Maybe we are trying to do too much for users here. Do you think this is really needed.

Reply:
This is just for the default, without it the prompt reads:

```sh
> lab
Enter GitLab URL (https://github.com):
```

Which is confusing and silly – we know for a fact that if anyone hits enter and supplies that, it will not work, so we shouldn't supply it as a default

Reply:
If they hit enter and they're not using hosted gitlab, it won't work either, but it has a way higher chance of working than trying to use GitLab's API against GitHub 🙃

Reply:
Also as above – bonus points for making it easier to know what you're meant to enter
- ah yes, the https url
- ah yes, not include the repository path, the root of the gitlab install